tests/contracts/chain-call-balance-1/output_1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "scilla_major_version": "0",
+  "gas_remaining": "7473",
+  "_accepted": "true",
+  "message": {
+    "_tag": "callC_accept",
```

Comment:
Where is this tag coming from? The message in `chain-call-balance-1/contract.scilla` is:
`m = { _tag : "b_accept_callC"; _amount : _amount; _recipient : addrB; addrC : addrC };`